handler/templates/handler.js

```diff
@@ -1,28 +1,29 @@
-import React from 'react';
-import {Resolver} from 'react-resolver';
+/** @flow */
+
+require('./styles.css');
```

Comment:
you should totally add css modules in here (with the other changes).

`import styles from './styles.css'`

and then `className={styles['<%= RouteName%>']}` down below